deps/asprintf/asprintf.c

```diff
@@ -7,54 +7,54 @@
 
 #ifndef HAVE_ASPRINTF
 
-#include <stdlib.h>
-#include <stdio.h>
 #include <stdarg.h>
+#include <stdio.h>
+#include <stdlib.h>
 
 #include "asprintf.h"
 
-int
-asprintf (char **str, const char *fmt, ...) {
-  int size = 0;
+int asprintf(char **str, const char *fmt, ...) {
   va_list args;
 
-  // init variadic argumens
+  // init variadic arguments
   va_start(args, fmt);
 
   // format and get size
-  size = vasprintf(str, fmt, args);
+  int size = vasprintf(str, fmt, args);
```

Comment:
What OS/compiler are you using where this caused a complication _problem_?

Reply:
I've used make 4.3 on archlinux system

Reply:
make && sudo make install

Reply:
Yes, that's being done in the `Dockerfile` without error.